more/free-programming-cheatsheets.md

```diff
@@ -80,6 +80,7 @@
 ### <a name="ide--editores"></a>IDE / Editors
 
 * [Editor VI - Guia de Referência](https://aurelio.net/curso/material/vim-ref.html) - Aurelio Marinho Jargas
+* [GNU Emacs Reference Card](https://www.gnu.org/software/emacs/refcards/pdf/refcard.pdf) - GNU.org (PDF)
```

Comment:
Please, could you explain this?

It's already merged at:
- https://github.com/EbookFoundation/free-programming-books/pull/5630

Reply:
I added the Emacs reference card resource in ide/editors category in more/free-programming-cheatsheets.md file.